hypervisor/arch/x86/configs/apl-up2.config

```diff
@@ -4,3 +4,9 @@ CONFIG_SERIAL_PCI=y
 CONFIG_SERIAL_PCI_BDF="0:18.0"
 CONFIG_COM_IRQ=6
 CONFIG_DMAR_PARSE_ENABLED=y
+CONFIG_HV_RAM_START=0x6e000000
```

Comment:
Are these suitable for both ABL and UEFI on the UP2 board?

Reply:
same question here

Reply:
Hi @gvancuts @binbinwu1 ,

Actually this address isn't used when relocation is on. So I will remove this to avoid confusion.